src/core/core.animations.js

```diff
@@ -1,121 +1,154 @@
 'use strict';
 
-var defaults = require('./core.defaults');
-var helpers = require('../helpers/index');
+const Animator = require('./core.animator');
+const Animation = require('./core.animation');
+const helpers = require('../helpers/index');
+const defaults = require('./core.defaults');
 
 defaults._set('global', {
 	animation: {
 		duration: 1000,
 		easing: 'easeOutQuart',
+		active: {
+			duration: 400
+		},
+		resize: {
+			duration: 0
+		},
+		numbers: {
+			type: 'number',
+			properties: ['x', 'y', 'borderWidth', 'radius']
```

Comment:
I was hesitant to add property type based fallback, but did not really test the impact on performance.

If that could be disabled in config, then I guess it would be ideal. I think this could be left for a later MR though.